packages/comms/src/ecl/workunit.ts

```diff
@@ -642,116 +642,118 @@
+                        case "bool":
+                            props[scopeProperty.Name] = !!+scopeProperty.RawValue;
+                            break;
+                        case "sz":
+                            props[scopeProperty.Name] = +scopeProperty.RawValue;
+                            break;
+                        case "s":
+                            props[scopeProperty.Name] = +scopeProperty.RawValue / 1000000000;
+                            break;
+                        case "ns":
+                            props[scopeProperty.Name] = +scopeProperty.RawValue;
+                            break;
+                        case "ts":
+                            props[scopeProperty.Name] = new Date(+scopeProperty.RawValue / 1000).toISOString();
+                            break;
+                        case "cnt":
+                            props[scopeProperty.Name] = +scopeProperty.RawValue;
+                            break;
+                        case "cost":
+                            props[scopeProperty.Name] = +scopeProperty.RawValue / 1000000;
+                            break;
+                        case "cpu":
+                        case "skw":
+                        case "node":
+                        case "ppm":
+                        case "ip":
+                        case "cy":
+                        case "en":
+                        case "txt":
+                        case "id":
+                        case "fname":
+                        default:
+                            props[scopeProperty.Name] = scopeProperty.RawValue;
                     }
+                    formattedProps[scopeProperty.Name] = formatNum(scopeProperty.Formatted ?? props[scopeProperty.Name]);
                 }
-                const dedup: DedupProperties = {};
-                for (const key in normalizedScope) {
-                    if (key.indexOf("__") !== 0) {
-                        const row = formatValues(normalizedScope, key, dedup);
-                        if (row) {
-                            normalizedScope.__groupedProps[row.Key] = row;
+                //  Other properties  ---
+            }
+            const normalizedScope: IScope = {
+                id: scope.Id,
+                name: scope.ScopeName,
+                type: scope.ScopeType,
+                Kind: scope["Kind"],
+                Label: scope["Label"],
+                __formattedProps: formattedProps,
+                __groupedProps: {},
+                ...props
+            };
+            if (normalizedScope[DEFINITION_LIST]) {
+                try {
+                    const definitionList = JSON.parse(normalizedScope[DEFINITION_LIST].split("\\").join("\\\\"));
+                    normalizedScope[DEFINITION_LIST] = [];
+                    definitionList.forEach((definition, idx) => {
+                        const matches = definition.match(definitionRegex);
```

Comment:
Kind of like all the others, seems like this isn't a problem because these are file paths. But I think it's specifically the `.*` part of `.*[\\\/]` that it's complaining about. And that's kind of overriding the exactly one or zero match of `([a-zA-Z]:)?` before it. Should be something like `[-_a-zA-Z0-9]*`, or whatever's valid for directory names I guess.